mingw-w64-cross-winpthreads/PKGBUILD

```diff
@@ -43,23 +39,60 @@ prepare() {
   autoreconf -vfi
 }
 
+_build() {
+  _target=$1
+  mkdir -p ${srcdir}/winpthreads-build-${_target} && cd ${srcdir}/winpthreads-build-${_target}
+  unset CC
+  ${srcdir}/mingw-w64/mingw-w64-libraries/winpthreads/configure \
```

Comment:
I am missing some sort of CRT selection. Wouldn't it pick the first found? Wouldn't the content of the `ucrt64`, `mingw32`, `mingw64` package variants be identical?

Reply:
From what I understand the `--host=${_target}` below makes it pick the right gcc/toolchain, which implies the crt etc.

Reply:
OK, it's probable it works like that.